packages/fluentui/react-northstar/src/components/Dialog/Dialog.tsx

```diff
@@ -221,15 +222,16 @@ export const Dialog: ComponentWithAs<'div', DialogProps> &
   });
 
   React.useEffect(() => {
+    const target = contentRef?.current;
```

Comment:
Just curious what is the difference between the old and new targets ?

Reply:
Previously were target to be added `overflow: hidden;` for this package the target is where it should still be possible to scroll.